auth.py

```diff
@@ -2,6 +2,7 @@
 import json
 import random
 import string
+from getpass import getpass
```

Comment:
```suggestion
# use the proper module for this
from getpass import getpass
```

Reply:
@otizonaizit What is ‘this’? Please be more descriptive